src/service/packet_router.proto

```diff
@@ -8,7 +8,7 @@ import "data_rate.proto";
 message packet_router_packet_up_v1 {
   bytes payload = 1;
   uint64 timestamp = 2;
-  uint32 rssi = 3;
+  sint32 rssi = 3;
```

Comment:
Add comment indicating units:
```suggestion
  // LoRa signal strength in dB.
  sint32 rssi = 3;
```

Reply:
should this be LoRa Signal Strength in `ddbi`?

Reply:
semtech protocol marks it as `dBm`(?)
https://github.com/Lora-net/packet_forwarder/blob/master/PROTOCOL.TXT#L144